refinedstorage-common/src/main/java/com/refinedmods/refinedstorage/common/detector/DetectorBlockEntity.java

```diff
@@ -165,7 +170,7 @@ public void updateActiveness(final BlockState state, @Nullable final BooleanProp
         super.updateActiveness(state, activenessProperty);
         final boolean powered = mainNetworkNode.isActive() && mainNetworkNode.isActivated();
         final boolean needToUpdatePowered = state.getValue(DetectorBlock.POWERED) != powered;
-        if (level != null && needToUpdatePowered && poweredChangeRateLimiter.tryAcquire()) {
+        if (level != null && needToUpdatePowered && poweredChangeTicks++ % POWERED_CHANGE_TICK_RATE == 0) {
```

Comment:
Yes and no. It will work fine till that value reaches MAX_INT. It's reached after running minecraft for 2147483647 ticks = 107.374.182,35 seconds = 1.789.569,707 minutes = 29.826,162 hours = 1.242,76 days = 3,4 years.
(The main reason that I didn't do it is because you didn't either: https://github.com/refinedmods/refinedstorage2/blob/develop/refinedstorage-common/src/main/java/com/refinedmods/refinedstorage/common/upgrade/UpgradeContainer.java#L182 but it does make sense to add it)